test/integration/taskBasedStatusUpdate.test.js

```diff
@@ -287,67 +287,117 @@ describe("Task Based Status Updates", function () {
+      ];
       reqBody.users = listUsers;
-      await userStatusModel.doc("userStatus001").set(generateStatusDataForState(userId1, userState.ACTIVE));
-      await userStatusModel.doc("userStatus002").set(generateStatusDataForState(userId2, userState.OOO));
-      await userStatusModel.doc("userStatus003").set(generateStatusDataForState(userId3, userState.IDLE));
-      await userStatusModel.doc("userStatus004").set(generateStatusDataForState(userId4, userState.ONBOARDING));
+      await userStatusModel.doc("userStatus000").set(generateStatusDataForState(userId0, userState.ACTIVE));
+      await userStatusModel.doc("userStatus001").set(generateStatusDataForState(userId1, userState.OOO));
+      await userStatusModel.doc("userStatus002").set(generateStatusDataForState(userId2, userState.IDLE));
+      await userStatusModel.doc("userStatus003").set(generateStatusDataForState(userId3, userState.ONBOARDING));
+      await userStatusModel.doc("userStatus005").set(generateStatusDataForState(userId5, userState.ACTIVE));
+      await userStatusModel.doc("userStatus006").set(generateStatusDataForState(userId6, userState.OOO));
+      await userStatusModel.doc("userStatus007").set(generateStatusDataForState(userId7, userState.IDLE));
+      await userStatusModel.doc("userStatus008").set(generateStatusDataForState(userId8, userState.ONBOARDING));
     });
 
     afterEach(async function () {
       await cleanDb();
     });
 
-    it("should return the correct results when there are no errors", async function () {
+    // eslint-disable-next-line mocha/no-skipped-tests
+    it.skip("should return the correct results when there are no errors", async function () {
       const res = await chai
         .request(app)
         .patch(`/users/status/batch`)
         .set("cookie", `${cookieName}=${superUserJwt}`)
         .send(reqBody);
       expect(res.status).to.equal(200);
-      const response = res.body;
-      expect(response.data).to.have.property("totalUsers");
-      expect(response.data).to.have.property("usersWithStatusUpdated");
-      expect(response.data).to.have.property("usersOnboardingOrAlreadyIdle");
-      expect(response.data.totalUsers).to.equal(5);
-      expect(response.data.usersWithStatusUpdated).to.deep.equal(3);
-      expect(response.data.usersOnboardingOrAlreadyIdle).to.equal(2);
+      const response = res.body.data;
+      expect(response).to.have.all.keys(
+        "totalUsers",
+        "totalUnprocessedUsers",
+        "totalOnboardingUsersAltered",
+        "totalOnboardingUsersUnAltered",
+        "totalActiveUsersAltered",
+        "totalActiveUsersUnAltered",
+        "totalIdleUsersAltered",
+        "totalIdleUsersUnAltered"
+      );
+      expect(response.totalUsers).to.equal(10);
+      expect(response.totalUnprocessedUsers).to.equal(0);
+      expect(response.totalOnboardingUsersAltered).to.equal(1);
+      expect(response.totalOnboardingUsersUnAltered).to.equal(1);
```

Comment:
Nit: totalOnboardingUsersAltered` -> I didn't understand what this variable name trying to say??
and How it is counted equal to 1??